provider/lxd/testing_test.go

```diff
@@ -552,16 +551,16 @@ func (conn *StubClient) StorageSupported() bool {
 	return conn.StorageIsSupported
 }
 
-func (conn *StubClient) StoragePool(name string) (api.StoragePool, error) {
-	conn.AddCall("StoragePool", name)
-	return api.StoragePool{
+func (conn *StubClient) GetStoragePool(name string) (pool *api.StoragePool, ETag string, err error) {
```

Comment:
is this flagged for  being replaced by a mockgen mock?

Reply:
Absolutely. I noted this in the prior PR - taking a short path with these to keep them "reviewable".